src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImpl.java

```diff
@@ -72,6 +72,20 @@
      */
     private DetailNode parent;
 
+    /**
+     * Reflects if any of the successors of this {@code DetailNode} has property
+     * {@link #unclosedTag} set true.
+     */
+    private Boolean hasUnclosedTags;
```

Comment:
it was resulting in violation about method with same name as field. I didn't feel that adding an exclusion would be for this class would be appropriate.